src/modules/header.less

```diff
@@ -2,6 +2,11 @@
     text-decoration: none;
 }
 
+#user-drawer-content {
+    max-height: 90vh;
+    overflow: auto;
+}
+
```

Comment:
@lnm95 Can you incorporate this change somehow? it makes the user drawer (popover when you click on the profile page) scrollable on zoomed in/small viewports. An issue reported by @TripleEYE

Reply:
Should be up at #11